### PR TITLE
General updates before the 14th release

### LINKED ISSUE #1984: Chip brightness error in the calculation of the opinionated theme
**Describe the bug**
In the documentation of the opinionated theme there's this info regarding the calculation of the Chip opinionated tokens:

image

But, if you go to the actual implementation, the percentage of lightness is not added, is subbed:

image

Which one is correct? We need to be consistent.

### PR DESCRIPTION
**Pull request by GomezIvann**
**Checklist**
- [x] The build process is done without errors. All tests pass in the `/lib` directory.
- [x] Self-reviewed the code before submitting.
- [x] Meets accessibility standards.
- [x] Added/updated documentation to `/website` as needed.
- [x] Added/updated tests as needed.

**Purpose**

**Description**
Several updates before releasing Halstack 14:
- **Removed DxcBox component from Halstack Design System.**
- `useToast` improved. The returned object is now memorized to avoid unnecessary renders inside components, hooks, etc.
- Included a toast inside the Halstack site to notify users about new available library versions.
- Theme generator updated with the most recent tokens. Also, the theme generator now uses our breadcrumbs instead of the custom ones.
- Fixed an issue with the data grid inside the theme generator.
- Theme page updates.
- Removed unnecessary `import React from "react";` from every affected file.
- Applied prettier on every file.
- Updated component status in the site.

**Closes #1984**

### CODE REVIEW

File: apps/website/screens/common/MainContent.tsx [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+  margin-inline: max(5%, 50% - 1124px/2);
+
+  @media (max-width: ${responsiveSizes.laptop}px) {
+    margin: 80px 32px;
+  }
+`;
+
+const MainContent = ({ children }: { children: ReactNode }) => {
+  const toast = useToast();
+  const router = useRouter();
```
Comment (Mil4n0r) [inline apps/website/screens/common/MainContent.tsx:19]: `router` is not required here

File: apps/website/screens/common/MainContent.tsx [hunk 2]
```diff
@@ -0,0 +1,64 @@
+import { useToast } from "@dxc-technology/halstack-react";
+import { ReactNode, useMemo, useState, useEffect } from "react";
+import styled from "styled-components";
+import { responsiveSizes } from "./variables";
+import { useRouter } from "next/router";
```
Comment (Mil4n0r) [inline apps/website/screens/common/MainContent.tsx:5]: `useRouter` is not required here

File: apps/website/screens/common/MainContent.tsx [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+            if (window) {
+              const currentUrl = window.location.href;
+              const newUrl = currentUrl.replace(/halstack\/\d+\//, `halstack/${latestRelease}/`);
+              window.location.href = newUrl;
+            }
+          },
+        },
+      });
+    }
+  }, [latestRelease, toast]);
```
Comment (Mil4n0r) [inline apps/website/screens/common/MainContent.tsx:59]: `pathVersion` should probably be added to the dependency array

File: apps/website/screens/components/tag/TagPageLayout.tsx [hunk 4]
```diff
@@ -21,6 +22,18 @@ const TagPageHeading = ({ children }: { children: React.ReactNode }) => {
             and multiple tags can be concatenated to generate a series of resources displayed with a visual hierarchy
             that calls the user&#39;s attention.
           </DxcParagraph>
+          <DxcAlert type="warning" size="fillParent">
+            This component will be removed from Halstack Design System in the next major release. Please, start
```
Comment (Mil4n0r) [inline apps/website/screens/components/tag/TagPageLayout.tsx:26]: I think this looks a bit better:

Please, start considering alternatives such as the `Badge` or `Chip` components.

File: apps/website/screens/components/toast/code/ToastCodePage.tsx [hunk 5]
```diff
@@ -292,9 +292,7 @@ const TextareaCodePage = () => {
   return (
```
Comment (Mil4n0r) [inline apps/website/screens/components/toast/code/ToastCodePage.tsx:292]: Function name has a typo.

`TextareaCodePage` -> `ToastCodePage`

File: apps/website/screens/theme-generator/themes/schemas/advanced.schema.json [hunk 6]
```diff
@@ -474,7 +479,7 @@
     "caretIconSpacing": "length",
     "borderRadius": "length",
     "borderStyle": "bStyle",
-    "borderThickness": "bWidth",
+    "borderThickness": "length",
```
Comment (Mil4n0r) [inline apps/website/screens/theme-generator/themes/schemas/advanced.schema.json:482]: Why is this changed?

**Review by Mil4n0r — CHANGES_REQUESTED** [review]

File: apps/website/screens/common/MainContent.tsx (hunk 2 above)
Comment (GomezIvann) [reply apps/website/screens/common/MainContent.tsx:5]: True!

File: apps/website/screens/common/MainContent.tsx (hunk 1 above)
Comment (GomezIvann) [reply apps/website/screens/common/MainContent.tsx:19]: Removed.

File: apps/website/screens/common/MainContent.tsx (hunk 3 above)
Comment (GomezIvann) [reply apps/website/screens/common/MainContent.tsx:59]: Totally true!

File: apps/website/screens/components/toast/code/ToastCodePage.tsx (hunk 5 above)
Comment (GomezIvann) [reply apps/website/screens/components/toast/code/ToastCodePage.tsx:292]: Done!

File: apps/website/screens/theme-generator/themes/schemas/advanced.schema.json (hunk 6 above)
Comment (GomezIvann) [reply apps/website/screens/theme-generator/themes/schemas/advanced.schema.json]: Already updated!

File: apps/website/screens/components/tag/TagPageLayout.tsx (hunk 4 above)
Comment (GomezIvann) [reply apps/website/screens/components/tag/TagPageLayout.tsx]: Done!

**Review by Mil4n0r — CHANGES_REQUESTED** [review]

**Review by Mil4n0r — APPROVED** [review]

**Pull request merged**